surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+- I have never wanted to contribute to the Rust Project. [NEXT](#for-non-contributors)
+
+> **justification**
+>
+> Fundamental for cohort analysis.
+
+## For active contributors
+
+### How did you get involved in the Rust Project?
+
+Select all that apply:
+
+- I wanted to improve my Rust skills and learn from the experts maintaining Rust.
+- I wanted to implement features that I needed as a user of Rust.
+- I wanted to give back to the Rust community.
+- I wanted to add to my portfolio of open source work for my resume.
+- I was encouraged to contribute by an existing Rust contributor / maintainer.
+- My work assigned open source contributions to me as part of my job.
+- Other: Free form
+
+> **justification**
+>
+> Useful in identifying paths into the project that are successful for future
+> contributors to follow.
+
+### How long have you been contributing to the Rust Project?
+
+- Less than 3 months
+- 3 to 6 months
+- 1 year
+- 2-3 years
+- 3-5 years
+- 5+ years
+
+> **justification**
+>
+> Useful for cohort analysis, the problems faced by contributors may differ
+> substantially for new contributors vs long time maintainers.
+
+### How are you associated with the Rust Project?
+
+- I am a member of a Rust Team.
+- I am a lead of a Rust Team.
+- I am a member of a Rust working group or project group.
+- I am a lead of a Rust working group or project group.
+- I am not a member of a Rust Team, working group, or project group, but I
+  would like to be.
+- I am not a member of a Rust team, working group, or project group, and I have
+  no intention to join one.
+
+> **justification**
+>
+> Cohort analysis. Some of the subsequent questions are only relevant to
+> certain cohorts.
+
+### What kind of work do you do within the Rust Project?
+
+Select all that apply:
+
+- contribution work
+- maintainership work
+- reviews
+- coordination
+- organization
+
+> **justification**
+>
+> Useful for cohort analysis. Problems will likely differ based on the work
+> being done. Those contributing new code will likely face diff problems than
+> those who mainly review or coordinate work.
+
+### What sort of work would you like to be focused on?
+
+Free Form.
+
+> **justification**
+>
+> Useful in identifying how satisified our contributors are with the work
+> they're currently doing or if we should be helping them branch out into new
+> areas / achieve new goals.
+>
+> This was one of the best received questions during the pre-survey interviews
+> and seemed to elicited thoughtful answers.
+
+### My experience contributing to the Rust Project so far has been positive.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### I feel positive about the project leadership and governance structure.
```

Comment:
We ask about overall project leadership and governance structure but we don't ask about overall health. Likewise we ask about the health of the contributor's specific working group or team but not about the governance structure and leadership. These points are related but in my mind distinct.

Reply:
good point, I'll add an equivalent question about health.